bucket/whisper-cpp.json

```diff
@@ -0,0 +1,40 @@
+{
+    "version": "1.2.1",
+    "description": "High-performance inference of OpenAI's Whisper automatic speech recognition (ASR) model",
+    "homepage": "https://github.com/ggerganov/whisper.cpp",
+    "license": "MIT",
+    "notes": [
+        "Whisper Model files are not included. You can download them here:",
+        "https://huggingface.co/ggerganov/whisper.cpp/tree/main"
+    ],
+    "bin": [
+        ["bench.exe", "whisper-bench"],
+        ["command.exe", "whisper-command"],
+        ["main.exe", "whisper"],
+        ["stream.exe", "whisper-stream"],
+        ["talk.exe", "whisper-talk"]
+    ],
```

Comment:
Is renaming really necessary? I'd prefer to leave them as is (except main.exe).

Reply:
I assumed that the original names would be too common, but we could just leave the names at their default and let the user resolve any name conflicts themselves.

Reply:
Yes, that's what I was thinking too...

Reply:
It has been done